db/citations.yml

```diff
@@ -2289,13 +2289,16 @@ bozer;aca22:
   grant: 'dmref19'
   journal: 'Acta Crystallogr. A'
   nb: ''
-  note: ''
+  note: 'arXiv:2204.00434v1 [cond-mat.mtrl-sci]'
   number: ''
   optannote: ''
   optwwwlanl: ''
   pages: ''
   title: 'Towards a machine-readable literature:
           finding relevant papers based on an uploaded powder diffraction pattern'
+  tags: 'first_submission_IUCr'
+  synopsis: ''
```

Comment:
you need to write these.... synopsis is for internal and needs to be short.  Something like:

a prototype application for finding published papers given a powder diffraction dataset is described.  This is an example of what might be possible if structured datasets are saved with papers in the future.

Reply:
Done